### ISSUE TITLE
[Settings]: All Text which is visually appearing as heading is not defined in heading programmatically.

### ISSUE THREAD

**Issue by v-rpundir** [microsoft/terminal#11246]
### Windows Terminal version (or Windows build number)

1.10.2383.0

### Other Software

**Test Environment:**
OS: Windows 11 Version 21H2 (OS Build 22000.168)
Screen Reader: Narrator
Tool: Accessibility Insight for Windows

### Steps to reproduce

**Repro Steps:**
Open Windows Terminal.
Open Settings page using "Ctrl + ',' ".
Open Narrator and ensure Scan mode is ON using 'Caps lock + Space' key.
Navigate through all tabs and verify that visually appearing heading is announced as heading or not by navigate through heading shortcut key 'h' or 'Shift + H'.

**User Experience:** 
Screen Reader user are impacted here as visually appearing heading is conveyed to them as heading as results, they will not experience same experience same structure which visual user is able.

**Guideline Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

**Attachments:**
[Text which is visually appearing as heading is not defined in heading programmatically..zip](https://github.com/microsoft/terminal/files/7175799/Text.which.is.visually.appearing.as.heading.is.not.defined.in.heading.programmatically.zip)

### Expected Behavior

All Text which is visually appearing as heading should be defined in heading programmatically. i.e. 'Startup' text which is visually appearing as heading should be defined as heading.

### Actual Behavior

All Text which is visually appearing as heading is not defined in heading programmatically. Issue repro throughout the Settings page. i.e. 'Startup' text which is visually appearing as heading is not defined as heading.
